packages/cf.js/src/provider.ts

```diff
@@ -214,4 +244,26 @@ export class Provider {
     }
     this.eventEmitter.emit(event.type, event);
```

Comment:
I would just replace the `break`s with `return this.eventEmitter.emit( ... )`

Reply:
Ah yea that does read nicer